### ISSUE TITLE
[Accessibility] ActivityUpdate: Voice Over does not convey the state for the 'Set due date' button under 'Activity update' screen.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6289]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33755064](https://microsoft.visualstudio.com/OS/_workitems/edit/33755064)

While navigating to the 'Set due date' button voice over does not announce the state as 'collapsed'. Voice over should convey the state for the 'Set due date' button as collapsed and when user activate it screen reader should convey the state as 'Set due date expanded'.

### Card JSON

[ActivityUpdate.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/ActivityUpdate.json)

**Comment by jwoo-msft** [comment]
iOS VO chimes when button state changes, and VO works with buttons accordingly.

**Comment by vagpt** [comment]
Closing this issue as per the above comment.
